CRM/Upgrade/Incremental/php/FiveFortyOne.php

```diff
@@ -65,6 +65,12 @@ public function setPostUpgradeMessage(&$postUpgradeMessage, $rev) {
   public function upgrade_5_41_alpha1($rev) {
     $this->addTask(ts('Upgrade DB to %1: SQL', [1 => $rev]), 'runSql', $rev);
     $this->addTask('Install legacy custom search extension', 'installCustomSearches');
+    $this->addTask('Replace legacy displayName smarty token in Invoice workflow template',
```

Comment:
This line is not part of this PR but it was merged & then lost when the line above was merged so this puts it back